dev/AppNotifications/AppNotificationContentBuilder/AppNotificationContentBuilder.idl

```diff
@@ -0,0 +1,148 @@
+﻿import "oaidl.idl";
+import "ocidl.idl";
+
+[contract(AppNotificationsContract, 1)]
+runtimeclass AppNotificationButtonBuilder
+{
+    Constructors
+    // Initializes a new instance of the ToastButton class.
+    ToastButton()
+
+    // Initializes a new instance of the ToastButton class.
+    ToastButton(String, String)
+
+
+    Properties
```

Comment:
Are most of these types strings/void functions?

Reply:
Most functions return self.